code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -711,4 +711,8 @@ datum/gear/uniform/fiendsuit
 
 datum/gear/uniform/fienddress
 	display_name = "Fiendish Dress"
-	path = /obj/item/clothing/under/fienddress
\ No newline at end of file
+	path = /obj/item/clothing/under/fienddress
+	
+datum/gear/uniform/leotard
```

Comment:
put / before
like 
/datum
intead of just datum

doesn't really matter but it'd be more consistent